### DISCUSSION TITLE
The following packages have unmet dependencies:

### DISCUSSION THREAD

**Discussion by SiLeNtKiLLeR-ArT** [projectdiscovery/naabu#772] [General]
go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest
github.com/google/gopacket/pcap
# github.com/google/gopacket/pcap
/root/go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap_unix.go:34:10: fatal error: pcap.h: No such file or directory
   34 | #include <pcap.h>
      |          ^~~~~~~~
compilation terminated.

**Comment by MetzinAround** [comment]
Hi! Thanks for the issue. 

I would recommend uninstalling naabu, then running `apt install libpcap-dev`, then reinstalling naabu. It shoudl work after that! :)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
